PDF_UA/2/8.9 Annotations/8.9.2 Semantics and content/8.9.2.4 Annotation types/8.9.2.4.13 Widget/verapdf-profile-8-9-2-4-13-t01.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.9 Annotations - 8.9.2.4 Annotation types - Widget annotation</name>
+        <description>A widget annotation of zero height and width, accordance with ISO 32000-2:2020, 12.7.5.5 shall be
+            an artifact</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDWidgetAnnot">
+            <id specification="ISO_14289_2" clause="8.9.2.4.13" testNumber="1"/>
+            <description>A widget annotation of zero height and width, accordance with ISO 32000-2:2020, 12.7.5.5 shall
+                be an artifact</description>
+            <test>width != 0 || height != 0 || structParentType == null || isArtifact == true</test>
+            <error>
+                <message>A Widget annotation of zero height and width is not an Artifact (structure tag = %1, standard type = %2)</message>
```

Comment:
A Widget annotation of zero height and width is not marked as an Artifact (structure tag = %1, standard type = %2)